docs/client/jdbc/kyuubi_jdbc.rst

```diff
@@ -127,34 +123,101 @@ accessing. The following is the format of the connection URL for the Kyuubi Hive
    - Properties are case-sensitive
    - Do not duplicate properties in the connection URL
 
-Connection URL over Http
+Connection URL over HTTP
 ************************
 
 .. versionadded:: 1.6.0
 
-.. code-block:: jdbc
+.. code-block::
 
-   jdbc:subprotocol://host:port/schema;transportMode=http;httpPath=<http_endpoint>
+   jdbc:kyuubi://host:port/schema;transportMode=http;httpPath=<http_endpoint>
 
 - http_endpoint is the corresponding HTTP endpoint configured by `kyuubi.frontend.thrift.http.path` at the server side.
 
 Connection URL over Service Discovery
 *************************************
 
-.. code-block:: jdbc
+.. code-block::
+
+   jdbc:kyuubi://<zookeeper quorum>/;serviceDiscoveryMode=zooKeeper;zooKeeperNamespace=kyuubi
+
+- zookeeper quorum is the corresponding zookeeper cluster configured by `kyuubi.ha.addresses` at the server side.
+- zooKeeperNamespace is the corresponding namespace configured by `kyuubi.ha.namespace` at the server side.
+
+Kerberos Authentication
+-----------------------
+
+.. versionadded:: 1.6.0
```

Comment:
this is just code refactoring? why do we need this?

Reply:
No, before 1.6.0, the Kerberos supports are based on UGI, as the Hive JDBC driver does, the following approaches are valid since 1.6.0.

Reply:
I can easily understand as the `Kerberos Authentication` is added in version 1.6.0

Reply:
Let me move the `.. versionadded::` to each approach to make it clear.